docs/source/applications/sextant.rst

```diff
@@ -0,0 +1,80 @@
+#########
+ Sextant
+#########
+
+TensorBay SDK supports methods to create evaluation by dataset.
```

Comment:
TensorBay SDK supports methods to interact with sextant application. which is better?